albumentations/augmentations/transforms.py

```diff
@@ -912,6 +912,53 @@ def get_transform_init_args_names(self):
                 'mask_value', 'approximate')
 
 
+class RandomGridShuffle(DualTransform):
+    """
+    Random shuffle grid's cells on image.
+
+    Args:
+        grid ((int, int)): size of grid for splitting image.
+
+    Targets:
+        image, mask
+
+    Image types:
+        uint8, float32
+    """
+
+    def __init__(self, grid=(3, 3), always_apply=False, p=1.0):
+        super(RandomGridShuffle, self).__init__(always_apply, p)
+        self.grid = grid
+
+    def update_params(self, params, **kwargs):
```

Comment:
not sure if we use `update_params`.. looks like you need `get_params` or `get_params_dependent_on_targets`